### PR TITLE
[WEB-1949] chore: delete workspace components restructuring.

### PR DESCRIPTION
**Pull request by prateekshourya29**
### Issue link: [WEB-1949](https://app.plane.so/plane/projects/02c3e1d5-d7e2-401d-a773-45ecba45d745/issues/d907addd-d30e-4e14-8851-f2d65955173c)

### CODE REVIEW

File: web/ce/components/workspace/delete-workspace.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  const [deleteWorkspaceModal, setDeleteWorkspaceModal] = useState(false);
+
+  return (
+    <>
+      <DeleteWorkspaceModal
+        data={workspace}
+        isOpen={deleteWorkspaceModal}
+        onClose={() => setDeleteWorkspaceModal(false)}
+      />
+      <Disclosure as="div" className="border-t border-custom-border-100">
```
Comment (SatishGandham) [inline web/ce/components/workspace/delete-workspace.tsx:28]: Can you use the collapsible component here.

File: web/core/components/workspace/settings/workspace-details.tsx [hunk 2]
```diff
@@ -308,44 +301,7 @@ export const WorkspaceDetails: FC = observer(() => {
...
-                        </Button>
-                      </div>
-                    </div>
-                  </Disclosure.Panel>
-                </Transition>
-              </div>
-            )}
-          </Disclosure>
-        )}
+        {isAdmin && <DeleteWorkspace workspace={currentWorkspace} />}
```
Comment (SatishGandham) [inline web/core/components/workspace/settings/workspace-details.tsx:304]: Give it a better name. DeleteWorkSpaceSection, maybe.

**Review by SatishGandham — CHANGES_REQUESTED** [review]

File: web/ce/components/workspace/delete-workspace.tsx (hunk 1 above)
Comment (prateekshourya29) [reply web/ce/components/workspace/delete-workspace.tsx]: I have added that change.

File: web/core/components/workspace/settings/workspace-details.tsx (hunk 2 above)
Comment (prateekshourya29) [reply web/core/components/workspace/settings/workspace-details.tsx]: Done!

**Review by SatishGandham — APPROVED** [review]

**Pull request merged**